contrib/minetest/template.py

```diff
@@ -34,17 +42,23 @@
 maintainer = "ttyyls <contact@behri.org>"
 license = "LGPL-2.1-or-later"
 url = "https://www.minetest.net"
-source = [
-    f"https://github.com/minetest/minetest/archive/refs/tags/{pkgver}.tar.gz",
-    f"https://github.com/minetest/irrlicht/archive/refs/tags/{_irrlichtver}.tar.gz",
-]
-source_paths = [".", "lib/irrlichtmt"]
-sha256 = [
-    "610c85a24d77acdc3043a69d777bed9e6c00169406ca09df22ad490fe0d68c0c",
-    "2fde8e27144988210b9c0ff1e202905834d9d25aaa63ce452763fd7171096adc",
-]
-tool_flags = {"CXXFLAGS": ["-Wno-deprecated-declarations"]}
+source = (
+    f"https://github.com/minetest/minetest/archive/refs/tags/{pkgver}.tar.gz"
+)
+sha256 = "070bc292a0b7fc60d7ff0a14b364c8229c5cbe38296a80f948ea2c2591545a5c"
 hardening = ["!int"]
+# fails to resolve the vendored irrlicht engine symlink
+options = ["brokenlinks"]
```

Comment:
this looks like a symlink that should be deleted instead of ignored

if it has to be there, use broken_symlinks instead